winsup/cygwin/include/cygwin/exit_process.h

```diff
@@ -49,9 +49,13 @@
 #endif
 
 static BOOL
-kill_via_console_helper (int bitness, wchar_t *function_name, int exit_code,
+kill_via_console_helper (HANDLE process, wchar_t *function_name, int exit_code,
                          DWORD pid)
 {
+  BOOL is_wow;
+  int bitness;
+  get_wow (process, is_wow, bitness);
```

Comment:
You should check the return value of get_wow I think

Reply:
Yes, if that fails this function shouldn't run, missed that.

Reply:
Done, https://github.com/msys2/msys2-runtime/pull/31/commits/099ee047406d7072d7e46c03939cd3cf9a9137d5